### PR TITLE
build(client.js): add apollo-link-error

### PR DESCRIPTION
**Pull request by NickyMeuleman**
Added apollo-link-error to the client setup for apollo

https://www.npmjs.com/package/apollo-link-error

### CODE REVIEW

**Comment by NickyMeuleman** [comment]
Hmmm, getting this warning on first load of the site. Navigation after that is fine.

chrome_2018-03-15_17-37-46

File: src/apollo/client.js [hunk 1]
```diff
@@ -39,7 +40,18 @@ const stateLink = withClientState({
...
+    graphQLErrors.map(({ message, locations, path }) =>
+      console.log(
+        `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`
+      )
+    )
+  }
+  if (networkError) console.log(`[Network error]: ${networkError}`)
+})
+
+const link = ApolloLink.from([stateLink, networkLink, errorLink])
```
Comment (pantharshit00) [inline src/apollo/client.js]: Move Error link to the first element. Ref https://github.com/apollographql/apollo-client/blob/a4548b0896073d6226709ea16d0ca875890943a4/packages/apollo-boost/src/index.ts#L80

Comment (NickyMeuleman) [reply src/apollo/client.js]: Done, great catch!

**Comment by NickyMeuleman** [comment]
Thank you @pantharshit00, that fixed it!

**Pull request merged**